### PR TITLE
Add features in Language-Version-History.md

### PR DESCRIPTION
**Pull request by leandromoh**

### CODE REVIEW

File: Language-Version-History.md [hunk 1]
```diff
@@ -111,11 +112,23 @@ Features Added in C# Language Versions
...
+- [User-defined conversion operators](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/user-defined-conversion-operators)
 - Properties
+- [Indexers](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/indexers/)
+- Output parameters (out and ref)
+- [Array trailing parameter (params)](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/params)
 - Delegates
 - Expressions
-- Statements
+- [Statements](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/statements-expressions-operators/statements)
+- [using statement](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/statements#the-using-statement)
```
Comment (leandromoh) [inline Language-Version-History.md:127]: there is no reference for `using` statement in the [statement link reference](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/statements-expressions-operators/statements)

**Comment by leandromoh** [comment]
@aekkasit114 does I need to do something else to be able to merge?

**Comment by 333fred** [comment]
@BillWagner can you please take a look?

File: Language-Version-History.md (hunk 1 above)
Comment (333fred) [inline Language-Version-History.md:127]: Doesn't seem like we need these specific statements, or we should remove the `Statement` bullet entirely.

File: Language-Version-History.md [hunk 2]
```diff
@@ -111,11 +112,23 @@ Features Added in C# Language Versions
...
 - Structs
+- Enums
 - Interfaces
 - Events
+- [Operator overloading](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/operator-overloading)
+- [User-defined conversion operators](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/user-defined-conversion-operators)
 - Properties
+- [Indexers](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/indexers/)
+- Output parameters (out and ref)
+- [Array trailing parameter (params)](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/params)
```
Comment (333fred) [inline Language-Version-History.md:123]:
```suggestion
- [`params` arrays](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/params)
```

`Array trailing parameter` is not a wording we've ever used.

File: Language-Version-History.md (hunk 1 above)
Comment (BillWagner) [inline Language-Version-History.md:127]: I would update these links as follows:

```suggestion
- [using statement](https://docs.microsoft.com/dotnet/csharp/language-reference/keywords/using-statement)
```

File: Language-Version-History.md [hunk 3]
```diff
@@ -111,11 +112,23 @@ Features Added in C# Language Versions
 # [C# 1.0](https://en.wikipedia.org/wiki/Microsoft_Visual_Studio#.NET_.282002.29) - Visual Studio .NET 2002
 - Classes
 - Structs
+- Enums
 - Interfaces
 - Events
+- [Operator overloading](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/operator-overloading)
```
Comment (BillWagner) [inline Language-Version-History.md:118]: In all cases, you should remove the `en-us`. (I'm only writing the suggestion on this instance). That way, the link will open in the reader's current culture:

```suggestion
- [Operator overloading](https://docs.microsoft.com/dotnet/csharp/language-reference/operators/operator-overloading)
```

File: Language-Version-History.md (hunk 1 above)
Comment (BillWagner) [reply Language-Version-History.md:127]: I'd recommend removing the "Statement" bullet.

Comment (leandromoh) [reply Language-Version-History.md:127]: > I'd recommend removing the "Statement" bullet.

Agree!

**Comment by leandromoh** [comment]
@333fred @BillWagner I will adjust as soon as possible!

**Comment by leandromoh** [comment]
@333fred @BillWagner adjusts and improviments done! :)

**Comment by 333fred** [comment]
@BillWagner for another review.

**Pull request merged**